build.gradle

```diff
@@ -35,6 +35,8 @@ ext {
           junitEngine          : 'org.junit.jupiter:junit-jupiter-engine:5.0.2',
           gcpCloudStorage      : 'com.google.cloud:google-cloud-storage:1.15.0',
           jCommander           : 'com.beust:jcommander:1.72',
+          openTracing          : 'io.opentracing:opentracing-api:0.31.0',
+          tracingJaeger        : 'com.uber.jaeger:jaeger-core:0.24.0',
```

Comment:
should the jaeger stuff be a separate artifact that clients can depend on if they choose to install jaeger? not sure where we draw line on what we package together. @mmihic @swankjesse any opinion?

Reply:
Yes, though I don’t think that’s necessary for this PR.

Reply:
How would this work when misk code has a references to classes in the jaeger artifact? Seems like you wouldn't really be able to separate the two, though there could be some Java/Kotlin sorcery I'm unaware of.

Reply:
You need to split the misk code and create a separate `misk-jaeger` artifact. if apps want jaeger, they include the `misk-jaeger` dep and then can install the `JaegerModule` in their app module.